influxdb_iox/tests/end_to_end_cases/management_cli.rs

```diff
@@ -669,6 +669,208 @@ async fn claim_database() {
+            .arg("release")
+            .arg(db)
+            .arg("--host")
+            .arg(addr)
+            .assert()
+            .success()
+            .stdout(predicate::str::contains(format!(
+                "Released database {}",
+                db
+            )))
+            .get_output()
+            .stdout
+            .clone(),
+    )
+    .unwrap();
+    let db_uuid = stdout.lines().last().unwrap().trim();
+
+    // delete the owner file <dir>/dbs/<uuid>/owner.pb
+    let mut owner_file: PathBuf = server_fixture.dir().into();
+    owner_file.push("dbs");
+    owner_file.push(db_uuid);
+    owner_file.push("owner.pb");
+
+    println!("Deleting {:?}", owner_file);
+    Command::new("rm")
+        .arg(owner_file.to_string_lossy().to_string())
+        .assert()
+        .success();
+
+    // Claiming db will now not work (no owner file)
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("database")
+        .arg("claim")
+        .arg(db_uuid)
+        .arg("--host")
+        .arg(addr)
+        .assert()
+        .failure()
+        .stderr(predicate::str::contains("owner.pb not found"));
+
+    // But does work when --force is supplied
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("database")
+        .arg("claim")
+        .arg(db_uuid)
+        .arg("--host")
+        .arg(addr)
+        .arg("--force")
+        .assert()
+        .success()
+        .stdout(predicate::str::contains(format!(
+            "Claimed database {}",
+            db_name
+        )));
+}
+
+#[tokio::test]
+async fn migrate_database_from_one_server_to_another() {
+    let server_fixture = ServerFixture::create_single_use(ServerType::Database).await;
+    let addr = server_fixture.grpc_base();
+
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("server")
+        .arg("set")
+        .arg("3113")
+        .arg("--host")
+        .arg(addr)
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("Ok"));
+
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("server")
+        .arg("wait-server-initialized")
+        .arg("--host")
+        .arg(addr)
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("Server initialized."));
+
+    let db_name = rand_name();
+    let db = &db_name;
+
+    // Create a database on one server
+    let stdout = String::from_utf8(
+        Command::cargo_bin("influxdb_iox")
+            .unwrap()
+            .arg("database")
+            .arg("create")
+            .arg(db)
+            .arg("--host")
+            .arg(addr)
+            .assert()
+            .success()
+            .stdout(predicate::str::contains("Created"))
+            .get_output()
+            .stdout
+            .clone(),
+    )
+    .unwrap();
+
+    let db_uuid = stdout.lines().last().unwrap().trim();
+
+    // figure out where the database lives and copy its data to a temporary directory.
```

Comment:
Along the same lines, I think in the future it would help to have documented *why* you might do this and that it's not really a production use case, something like:

```suggestion
    // figure out where the database lives and copy its data to a temporary directory,
    // as you might copy data from remote object storage to local disk for debugging.
```